addon/components/rdf-input-fields/input-field.js

```diff
@@ -126,8 +134,63 @@ export default class InputFieldComponent extends Component {
       })
     );
   }
+
+  findFieldOption(propertyName, predicate, returnType = 'string') {
```

Comment:
Change `propertyName, predicate` to one parameter and check on isNamedNode or typeof string.

+ add comment that this is done for backwards compatibility

Reply:
Done by using this getter
```js
  getOptionPredicates() {
    return {
      conceptScheme: EXT('conceptScheme'),
      searchEnabled: EXT('searchEnabled'),
    };
  }